### PR TITLE
Fix #131 Add starter version of the Contributor Introduction Script

### LINKED ISSUE #131: "Introduction" Script
We should make a script for the "Introduction" video in the "Contributor" section of the learning path.  The script should be submitted via pull request.

### PR DESCRIPTION
**Pull request by lenucksi**
Here's an initial version of a script for the introduction video of the Contributor section.
Happy about any reviews and proposals to shorten the text.

### CODE REVIEW

File: contributor/01-introduction-script.md [hunk 1]
```diff
@@ -8,5 +8,80 @@
...
+-> Flip open the TC role name on the slide
+
+X: A Trusted Committer will be your host for your stay in the hosting community.
+They are the gatekeepers to the project's code repository and will move your contribution closer to production once they accepted it. 
+It is their role to mentor you on your way to contributing to their community, e.g. by providing good information to enable you to help yourself or help you in other cases.
+
+-> Flip open the PO role name on the slide
+
+X: Your interaction with the Product Owner will likely focus on working out the fit of your contribution into the general product and its road map. 
+You might work this role to make sure that general aspects of documentation or consistency of UI/UX are kept upon merging your contribution.
```
Comment (MaineC) [inline contributor/01-introduction-script.md]:
```suggestion
You might work with this role to make sure that general aspects of documentation or consistency of UI/UX are kept upon merging your contribution.  
```

File: contributor/01-introduction-script.md [hunk 2]
```diff
@@ -8,5 +8,80 @@
...
+### Section overview
+
+-> Possbly slide with the segment names, popping up one after another?
+
+[EDITOR NOTE J:
+I suggest alternating between segments.
+Also, possibly shorten the texts.
+]
+
+X: In the following 5 videos you will learn more in detail about the various aspects introduced before.
```
Comment (MaineC) [inline contributor/01-introduction-script.md]:
```suggestion
X: In the following 5 videos you will learn more details about the various aspects introduced before. 
```

File: contributor/01-introduction-script.md [hunk 3]
```diff
@@ -8,5 +8,80 @@
...
+X: In the following 5 videos you will learn more in detail about the various aspects introduced before. 
+The second segment, i.e. the one after this segment, will detail _mindset and habit_ that create opportunities to _become an InnerSource Contributor_.
+
+X: In the third segment we'll look into the _ethos of the Contributor_ - i.e. aspects of _behavior_ that will lead to a pleasant and productive time for you and the host team and might spark more collaboration.
+The guest-in-home analogy presented in the introductory videos will serve as a vivid example. 
+
+X: The fourth segment describes the hands-on things to do to make your contribution a success - the _mechanics of Contributing_.
+We'll give practical tips to leverage when preparing to work on a contribution, during development, and also in the pull request.  
+
+X: After we've dealt with the personal interaction-focused and technical of the contributor role, the fifth segment presents the benefits of making the effort to contribute from various perspectives: Your perspective, your team's perspective, and the company's perspective at large.
```
Comment (MaineC) [inline contributor/01-introduction-script.md]:
```suggestion
X: After we've dealt with the personal, the interaction-focused, and the technical aspects of the contributor role, the fifth segment presents the benefits of making the effort to contribute from various perspectives: Your perspective, your team's perspective, and the perspective of the company at large.
```

File: contributor/01-introduction-script.md [hunk 4]
```diff
@@ -8,5 +8,80 @@
...
+
+X: The last segment will recap what we've learned about being an InnerSource contributor.  We'll share how you can continue your learning on InnerSource both with with other online videos and also involvement with the online InnerSource community.
```
Comment (MaineC) [inline contributor/01-introduction-script.md]:
```suggestion
X: The last segment will recap what we've learned about being an InnerSource contributor.  We'll share how you can continue your learning on InnerSource both with other online videos and involvement with the online InnerSource community.
```

**Comment by lenucksi** [comment]
Thanks for the reviews, merging this then. Please add additional changes to a new PR should they come up.

**Pull request merged**